src/Resources/skeleton/resetPassword/twig_email.tpl.php

```diff
@@ -4,6 +4,6 @@
 
 <a href="{{ url('app_reset_password', {token: resetToken.token}) }}">{{ url('app_reset_password', {token: resetToken.token}) }}</a>
 
-<p>This link will expire in {{ tokenLifetime|date('g') }} hour(s).</p>
+<p>This link will expire in {{ resetToken.getExpirationMessageKey()|trans(resetToken.getExpirationMessageData(), 'ResetPasswordBundle') }}.</p>
```

Comment:
Fun fact: we need to check the version of the bundle in question (e.g. reset-password here) and either:

A) Toggle 2 "modes" of generated code based on the version
B) Throw an exception and tell people to upgrade to version 1.WHATEVER

Option (B) seems harsh, but I think it's probably a good idea. It's an opportunity for us to notify them that a newer version is available that fixes a bug we know about. And, it will probably be rare (as users will typically install the bundle right before running this command.

Reply:
```suggestion
<p>This link will expire in {{ resetToken.expirationMessageKey|trans(resetToken.expirationMessageData, 'ResetPasswordBundle') }}.</p>
```

Reply:
done